cmd/debos/debos.go

```diff
@@ -144,13 +144,13 @@ func main() {
 		return
 	}
 
-	/* If fakemachine is supported the outer fake machine will never use the
+	/* If fakemachine is enabled the outer fake machine will never use the
```

Comment:
Unsure how to comment on a whole commit rather then change lines. 

But this commit doesn't seem to belong in this PR anyhow. And the commit message doesn't explain the rationale for the change either. So please seperate it and explain why the behaviour change.

Reply:
agree, i will move that commit to a different PR